content/en/users/getting-started/architecture/_index.md

```diff
@@ -113,72 +105,83 @@ configuration provided at the
 The integration relies on the OpenStack Keystone
 [OS-FEDERATION API](https://developer.openstack.org/api-ref/identity/v3-ext/index.html#os-federation-api).
 
-#### Legacy VOMS / X.509 certificates
+#### Legacy X.509 certificates
+
+EGI can support users still using X.509 certificates extended with VO
+attributes (e.g. acknowledging that the user is member of the VO), also
+known as Virtual Organization Membership Service (VOMS) proxy certificates.
+VOMS proxy certificates are used in subsequent calls to the endpoints, which
+map the certificate and VO information via specific integration modules,
+allowing authentication and authorization using X.509 certificates.
 
-EGI can support users still using X.509 certificates extended with VO attributes
-(e.g. acknowledging that the user is member of the VO) in a so called VOMS
-proxy. This VOMS proxy certificate is used in subsequent calls to the endpoints
-which map the certificate and VO information via specific integration modules
-for VOMS authentication.
+{{% alert title="Note" color="info" %}} VOMS proxy certificates can be obtained
+from user certficates or from other proxy certificates, and are useful in
+multi-step delegation of user roles.
+{{% /alert %}}
 
 There are two implementations for the support of VOMS proxies:
 
 - [KeyStorm](https://github.com/the-rocci-project/keystorm) provides federated
   authentication for OpenNebula/rOCCI.
 - [Keystone-VOMS](https://github.com/IFCA/Keystone-VOMS) is an OpenStack
-  Keystone plugin to enable VOMS authentication. It allows users to get tokens
-  which can be used to access any of the OpenStack services. Users are generated
-  on the fly in Keystone, it does not need regular synchronization with the VO
-  Management server Perun.
+  [Keystone](https://docs.openstack.org/keystone/latest/) plugin to enable VOMS
+  authentication. It allows users to get tokens which can be used to access any
+  of the OpenStack services. Users are generated on the fly in Keystone, it
+  does not need regular synchronization with the VO Management server Perun.
```

Comment:
I had to think twice why Perun was mentioned here, this is quite legacy. I would just remove everything from from the first period onwards.

Reply:
I just took that page as it is currently is under cloud compute, made some revisions to it (but id not remove any significant part). maybe we can do a call and revise it together.